pkg/scraper/script.go

```diff
@@ -30,6 +30,13 @@ func newScriptScraper(scraper scraperTypeConfig, config config, globalConfig Glo
 func (s *scriptScraper) runScraperScript(inString string, out interface{}) error {
 	command := s.scraper.Script
 
+	if command[0] == "python" {
```

Comment:
this should work both ways (some users have already edited the scraper's yaml file to use python3)
```golang
	if command[0] == "python" || command[0] == "python3" {
```